main.tf

```diff
@@ -1,15 +1,17 @@
 provider "azurerm" {
-  version = "1.22.1"
+  features {}
```

Comment:
Add AzureRm version

version = "=2.20.0"

Reply:
Added version as per review comments

Reply:
Done the changes as per review comments, also updated version to the latest 29.0